### PR TITLE
Remove unused bucket-setup and notification-configuration lambdas

### PR DESCRIPTION
**Pull request by sir-sigurd**

### CODE REVIEW

**Review by akarve — APPROVED** [review]

**Comment by akarve** [comment]
weird that catalog tests failed; plz wait for approval from @kevinemoore before merge as he knows these codepaths better

**Review by kevinemoore — APPROVED** [review]

**Pull request merged**
